packages/framework/src/Form/Admin/Administrator/AdministratorFormType.php

```diff
@@ -35,11 +33,13 @@ class AdministratorFormType extends AbstractType
      * @param \Symfony\Component\Security\Core\Security $security
      * @param \Shopsys\FrameworkBundle\Model\Administrator\RoleGroup\AdministratorRoleGroupFacade $administratorRoleGroupFacade
      * @param \Shopsys\FrameworkBundle\Model\Security\Roles $roles
+     * @param \Shopsys\FrameworkBundle\Component\Router\Security\RouteCsrfProtector $routeCsrfProtector
      */
     public function __construct(
         private readonly Security $security,
         private readonly AdministratorRoleGroupFacade $administratorRoleGroupFacade,
         private readonly Roles $roles,
+        protected readonly RouteCsrfProtector $routeCsrfProtector,
```

Comment:
```suggestion
        private readonly RouteCsrfProtector $routeCsrfProtector,
```

just because others are private too, and form types are not meant to be extended